src/main/java/lotto/prize/LottoPrize.java

```diff
@@ -0,0 +1,50 @@
+package lotto.prize;
+
+import lotto.money.Money;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public enum LottoPrize {
+    FIRST(6, Money.from(2_000_000_000)),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```

생성자에서 int -> Money 역할을 해준다면 Money.from의 중복을 제거할 수 있을 것 같네요 :)

Reply:
아하! 그렇네요 감사합니다 :)